python/lsst/sims/catUtils/mixins/AstrometryMixin.py

```diff
@@ -92,6 +94,11 @@ class AstrometryGalaxies(AstrometryBase):
     This mixin contains a getter for the corrected RA and dec which ignores parallax and proper motion
```

Comment:
This is confusing to me. Doesn't anything that calls _observedFromICRS like the 2nd and 3rd getter in this mixin take these effects into account?